### ISSUE TITLE
🎨 [App Layout] Create a responsive website header

### ISSUE THREAD

**Issue by LemmyMwaura** [italanta/elewa-website#3]
#  Create the website header
---

#### Note: This issue is dependent on #2 , please complete that before you continue.

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user I want a header I can use to navigate the website.

### Description

Your task is to add the website header as described below.

The content of this header will include:

```
1. "Elewa" logo on the left
2. A transparent background 
3. White text
```

The header needs to be absolute and of higher z-index so that it appears on top of the page banner.

The header needs to appear as shown below on web (the background color should be transparent as it will be re-used across the different pages):

![banner](https://github.com/italanta/elewa-website/assets/20738858/0a663caf-2a8b-42b0-80cd-c668157086eb)

### Technical Analysis

Create a layout package in libs/elements/layout

Create a header component inside a components folder of the package above

### Acceptance Criteria

1. When rendered the component is pixel perfect compared to the design above.

**Comment by jrosseel** [comment]
Closed as duplicate of #1
